nextstrain/cli/command/login.py

```diff
@@ -0,0 +1,127 @@
+"""
+Logs in a Nextstrain.org Groups user, storing their credentials locally.
+"""
+import jose
+import getpass
+from pycognito import Cognito
+from typing import Dict
+from .. import config
+from ..errors import UserError as NextstrainUserError
+
+
+COGNITO_CLIENT_ID='2vmc93kj4fiul8uv40uqge93m5'
+COGNITO_USER_POOL_ID='us-east-1_Cg5rcTged'
+
+
+def register_parser(subparser):
+    parser = subparser.add_parser("login", help = "Login as a Nextstrain.org Groups user")
+
+    register_arguments(parser)
+
+    return parser
+
+
+def register_arguments(parser):
+    parser.add_argument(
+        "--check",
+        help    = "Check credentials by logging into nextstrain.org",
+        action  = 'store_true')
+
+    return parser
+
+
+def run(opts):
+    if opts.check:
+        print("Checking credentials by logging into nextstrain.org...")
+        username = config.get('login', 'username', fallback=None, path=config.SECRETS)
+
+        if not username:
+            login()
+            return 0
+
+        tokens = {
+            k : config.get('login', k, fallback=None, path=config.SECRETS)
+                for k in [ 'access_token', 'id_token', 'refresh_token' ]
+        }
+
+        u = Cognito(COGNITO_USER_POOL_ID, COGNITO_CLIENT_ID, username=username,
+                id_token=tokens['id_token'],
+                refresh_token=tokens['refresh_token'],
+                access_token=tokens['access_token'])
+
+        try:
+            u.verify_token(tokens['id_token'], 'id_token', 'id')
+            u.verify_token(tokens['access_token'], 'access_token', 'access')
+
+        except (jose.exceptions.JWSError, jose.exceptions.JWTError):
```

Comment:
I think we can avoid knowing the implementation detail that `pycognito` uses `jose` and instead catch `pycognito.TokenVerificationException`. This also means we don't have to explicitly dep or import `jose`.